os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -74,6 +75,7 @@ void *rtl_set_ns_func(void)
 	ns_func.flash_write = ns_flash_write;
 	ns_func.device_lock = device_mutex_lock;
 	ns_func.device_unlock = device_mutex_unlock;
+	ns_func.setstatusbits = NULL;
```

Comment:
If you have a  plan that implement setstatusbits, then put some TODO comment, otherwise get rid of this function from nsfunc_ops_s and here.

Reply:
It have been implemented, but due to this PR is blocking it cannot be merge.
[#PR4798](https://github.com/Samsung/TizenRT/pull/4798) is dependent on this PR.